docs/changes.rst

```diff
@@ -119,7 +119,7 @@ Released on October 15th 2019.
 * Display addon parameters on overview.
 * Sentry is now supported through modern Sentry SDK instead of Raven.
 * Changed example settings to be better fit for production environment.
-* Added automated backups using Borg backup.
+* Added automated backups using BorgBackup.
```

Comment:
Good point, fixed that in rest of the docs in 8d35a1ff489c358f7f2772f5aba4ee474ed3ff39.